source/release-notes.txt

```diff
@@ -10,6 +10,27 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.42.1
+----------------
+
+*Released February 15, 2024*
+
+New Features
+
+- Added vector search index creation to create search indexes 
+  modal :issue:`COMPASS-7302`.
+- Vector search type indexes now display in the search index table 
+  :issue:`COMPASS-7509`.
+
+Bug Fixes
+
+- Improved validation of command line arguments :issue:`COMPASS-7260`.
```

Comment:
I dropped the mention of intercom / active user information in the [Github Release Notes](https://github.com/mongodb-js/compass/releases/tag/v1.42.1) as this seemed like a internal concept. LMK if you disagree.